test/unit/pusherConstructorTest.php

```diff
@@ -78,22 +78,53 @@ public function testEncryptedOptionWillBeOverwrittenByHostAndPortOptionsSetHostA
 				'port' => '3000'
 			);
 			$pusher = new Pusher( 'app_key', 'app_secret', 'app_id', $options );
-			
+
 			$settings = $pusher->getSettings();
 			$this->assertEquals( 'http', $settings[ 'scheme' ] );
 			$this->assertEquals( $options[ 'host' ], $settings[ 'host' ] );
 			$this->assertEquals( $options[ 'port' ], $settings[ 'port' ] );
-		}	
-		
+		}
+
 		public function testSchemeIsStrippedAndIgnoredFromHostInOptions() {
 			$options = array(
 				'host' => 'https://test.com'
 			);
 			$pusher = new Pusher( 'app_key', 'app_secret', 'app_id', $options );
-			
+
 			$settings = $pusher->getSettings();
 			$this->assertEquals( 'http', $settings[ 'scheme' ] );
 			$this->assertEquals( 'test.com', $settings[ 'host' ] );
-		}		
+		}
+
+    public function testClusterSetsANewHost() {
```

Comment:
You could include a linter in the travis build. The `new-lib` branch already does this, you could use that as an example.